pkg/metrics/client_go_adapter.go

```diff
@@ -0,0 +1,266 @@
+/*
+Copyright 2016 The Kubernetes Authors.
```

Comment:
s/2016/2018 (someone should build github suggestion bot for this :))

Reply:
ack